themes/bootstrap5/scss/components/search.scss

```diff
@@ -391,6 +391,10 @@ header .container.navbar { margin-bottom: 0; }
 
 .result-links a {
   line-height: 1.5;
+
+  &.icon-link {
+    display: flex;
```

Comment:
In practice this seems not to be actually necessary; the links wrap anyway due to the narrow column width.  But custom templates might mess with that so if the intention is that they wrap, we should make that explicit.